geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/NullRedisSortedSet.java

```diff
@@ -86,6 +86,11 @@ long zcount(SortedSetScoreRangeOptions rangeOptions) {
     return increment;
   }
 
+  @Override
+  List<byte[]> zpopmin(Region<RedisKey, RedisData> region, RedisKey key, int count) {
```

Comment:
I think this should go below "zpopmax" alphabetically.

Reply:
Done